tests/fiaas_deploy_daemon/specs/v3/test_v3factory.py

```diff
@@ -83,6 +83,7 @@
         "ingresses[0].pathmappings[0].path": "/",
         "ingresses[0].pathmappings[0].port": 80,
         "secrets_in_environment": False,
+        ""
```

Comment:
I might be missing something, but I don't think this has any effect in practice. Is it a typo?